File: Pattern_Printing/Java/Print_the_pattern_Set-1.java
class GfG {
    void printPat(int n) {
        // Your code here
        for (int k=0; k<n; k++) {
            for (int i=n; i>=1; i--) {
                for (int j=1; j<= n-k; j++) {
                    System.out.print(i + " ");
               }
            }
            System.out.print("$");
        }
    }
}

File: pattern-printing/print-the-pattern-set/print-the-pattern-set_1.cpp
class Solution {
  public:
    vector<int> printPat(int n) {
        // write code here
        vector<int> ans;

        for(int row = n; row > 0; row--) {
            for(int num = n; num > 0; num--) {
                for(int times = 1; times <= row; times++) {
                    ans.push_back(num);
                }
            }
            ans.push_back(-1);
        }
        return ans;
    }
};